README.md

```diff
@@ -1,4 +1,4 @@
-# @mattermost/react-native-turbo-log
+# [@mattermost](https://github.com/mattermost)/[react-native-turbo-log](https://github.com/mattermost/react-native-turbo-log)
```

Comment:
Not sure why you want to add the links that way

Reply:
As this line was in the format of `Organization\Repository`, so I thought of adding separate links for the org and the repo. @enahum Do you think it will be better to keep same link for that line which redirects to the repo?

Reply:
This is really the name of the npm package, and the link would be pointing to the same root of this sane repo, so I guess no changes are needed here, same as the other libraries we've built.

What do you think?

Reply:
Ok @enahum then its better to keep it as it was.